plugins/inputs/huawei_routers_telemetry/huawei_routers_telemetry_test.go

```diff
@@ -0,0 +1,27 @@
+
+	grouper, _ := HuaweiTelemetryDecoder(packet)
+	acc.AddMetric(grouper.Metrics()[0])
+
+	require.Empty(t, acc.Errors)
+
+	tags := map[string]string{"path": "huawei-devm:devm/cpuInfos/cpuInfo", "position": "\"9\"", "source": "HuaweiTest1", "subscription": "ProbandoTelemetriaSubscripcion"}
+	fields := map[string]interface{}{"entIndex": uint32(17367041), "interval": uint32(8), "ovloadThreshold": uint32(90), "systemCpuUsage": uint32(10), "unovloadThreshold": uint32(75)}
+	acc.AssertDoesNotContainsTaggedFields(t, "huawei-devm:devm/cpuInfos/cpuInfo", fields, tags)
```

Comment:
Shouldn't you test for the actual metric instead of asserting on something is not contained? How about
```suggestion
	expected := []telegraf.Metric{
		testutil.MustMetric(name, tags, fields),
	}
	testutil.RequireMetricsEqual(t, metrics, expected)
```